### PR TITLE
fix(types): Add missing silent option

### PR DESCRIPTION
**Pull request by negezor**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [ ] Have you added new tests to prevent regressions?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [X] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/master/CONTRIBUTING.md)?

### Description of change
Add missing `silent` option

### CODE REVIEW

**Comment by SimonSchick** [comment]
Add a super small test pls

File: types/test/usage.ts [hunk 1]
```diff
@@ -15,6 +15,15 @@ async function test(): Promise<void> {
 
     user = await User.findOne();
 
+    if (!user) {
```
Comment (sushantdhiman) [inline types/test/usage.ts]: This is required?

Comment (negezor) [reply types/test/usage.ts]: Method findOne may return null.

Comment (sushantdhiman) [reply types/test/usage.ts]: So typescript gives you warning/error without this?

Comment (negezor) [reply types/test/usage.ts]: Screenshot

Comment (SimonSchick) [reply types/test/usage.ts]: @sushantdhiman fyi this will not error when specifyin` rejectOnEmpty`, TS is got the smarts :)

Comment (sushantdhiman) [reply types/test/usage.ts]: :+1:

**Pull request merged**